roles/transformers/vars/CentOS7.yml

```diff
@@ -1,5 +1,5 @@
 ---
-imagemagick_version: "{{ dependencies_version.imagemagick }}-ci"
+imagemagick_version: "{{ dependencies_version.imagemagick }}-ci-{{ dependencies_version.imagemagick_release }}"
```

Comment:
I still think that just using`dependencies_version.imagemagick` would be simpler and gives more flexibility on the package version